source-code-flow-manager-gitlab/src/main/java/com/iprody/source/code/flow/manager/branch_api/BranchApi.java

```diff
@@ -0,0 +1,41 @@
+package com.iprody.source.code.flow.manager.branch_api;
+
+import com.iprody.source.code.flow.manager.branch_api.entity.Branch;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+public class BranchApi {
+
+    private final String PRIVATE_TOKEN = "glpat-VwpTqGWV-SSQwV7xUypq";
+//    private final String BASE_URL = "https://gitlab.com/api/v4/projects/";
+
+    private WebClient webClient;
+
+    public BranchApi(String baseUrl) {
+        this.webClient = WebClient.create(baseUrl);
+    }
+
+    public Mono<Branch> createBranch(int projectId, String branchName, String parentBranchName) {
```

Comment:
Please keep naming same as Gitlab API express it. It will help to understand what these parameter mean as in a request to Gitlab API
```suggestion
    public Mono<Branch> createBranch(int projectId, String branch, String ref) {
```

Reply:
Ok. I'm using GitLab API naming.